partner_communication_switzerland/models/contracts.py

```diff
@@ -402,8 +402,14 @@ def _send_welcome_active_letters_for_activated_sponsorships(self):
             ]
         )
         if to_send:
```

Comment:
This line can be removed, if no records complies the search criterion, there will be no record to iterate on and we won't even enter the `for` loop.